src/base/PrettyPrinters.ml

```diff
@@ -108,7 +108,7 @@ let scilla_error_to_json elist =
   let err_to_json (e : scilla_error) =
     `Assoc
       [
-        ("error_message", `String e.emsg);
+        ("error_message", `String (mk_error_description e));
```

Comment:
Would it be a good idea to separate `kind` and `instance` here in the JSON too? The only downside would be that the blockchain code which uses this may need to be modified too.

Reply:
Good suggestion. Could we do this in a separate PR so this one is not too intrusive?

Reply:
> Good suggestion. Could we do this in a separate PR so this one is not too intrusive?

definitely, yes